karaf/images/centos/run-java.sh

```diff
@@ -33,7 +33,7 @@
 #                             https://www.youtube.com/watch?v=w1rZOY5gbvk
 #                             https://vimeo.com/album/4133413/video/181900266
 # Also note that heap is only a small portion of the memory used by a JVM. There are lot
-# of other memory areas (metadata, thread, code cache, ...) which addes to the overall
+# of other memory areas (metadata, thread, code cache, ...) which adds to the overall
```

Comment:
How was this file updated ? Normally you should update karaf/fish-pepper.yml, too with the proper version for run-java and then re-run fish-pepper.

Reply:
I run fish-pepper in _karaf_ directory. In 526cdb3a07b0767be2681792d587c7af4a800dea versions are updated in the karaf directory but I assume fish-pepper didn't run. The files in the _java_ directory has been updated.

Reply:
👍